### DISCUSSION TITLE
[Question] Is it safe to be using 053h?

### DISCUSSION THREAD

**Discussion by puelp** [celzero/rethink-app#560] [Q&A]
I am currently on 053h because I like the bulk-blocking features offered by 053h that is missing on 053i/053j.

Is that OK? Or am I exposing myself to security risks in 053h that were fixed by 053i/053j?

**Comment by ignoramous** [comment]
We are not aware of any leaks either in `v053h` or in `v053i`, but there could be as Rethink isn't an app that's been audited for security. The userbase is tiny, really, to spend any money on security audits for what's an open source and libre software. May be, someday.

> I am currently on 053h because I like the bulk-blocking features offered by 053h that is missing on 053i/053j.

We are bringing back bulk apply firewall rules #524
